draft/spec/index.html

```diff
@@ -698,6 +699,14 @@ <h2>Version</h2>
                             containing <a>logical path</a> names of files in the OCFL Object state that have
                             content with the given digest.
                         </p>
+                        <p>
+                            <a href="#logical-path">Logical paths</a> present the structure of an OCFL Object at a
+                            given version. This is given as an array of file paths, with the following restrictions
```

Comment:
consider "This is given as an array of values, with the following restrictions to provide for path safety in the common case of the logical path value representing a file path."

Reply:
Nice wording. I think we have to be strict to cover that case but I agree that it may not be used in that way

Reply:
At that point aren't we just obfuscating?

Reply:
I have incorporated Julian's suggested revision into the latest version